src/document-state.js

```diff
@@ -112,6 +116,32 @@ export class DocumentState {
   onVisibilityChanged_() {
     this.visibilityObservable_.fire();
   }
+
+  /**
+   * @param {function()} handler
```

Comment:
Good to know, thanks! Changing that here and in `waitForChild` would require fixing several unit tests. I'd be happy to make that change if you'd like.